install/v0.1.0/kfserving.yaml

```diff
@@ -0,0 +1,633 @@
+apiVersion: v1
```

Comment:
should we follow the knative release way to upload this yaml to the GitHub release artifact?

Reply:
Yeah -- once these are checked in, I'll absolutely add them as release artifacts.